tests/algorithms/test_preference_comparisons.py

```diff
@@ -290,6 +292,57 @@ def build_preference_comparsions(gatherer, reward_trainer, fragmenter, rng):
     build_preference_comparsions(None, None, random_fragmenter, rng=rng)
 
 
+@patch("builtins.input")
+@patch("IPython.display.display")
+def test_synchronous_human_gatherer(mock_display, mock_input):
+    del mock_display  # unused
+    gatherer = preference_comparisons.SynchronousHumanGatherer(
+        video_dir=pathlib.Path("."),
+    )
+
+    # these inputs are designed solely to pass the test. they aren't tested for anything
+    trajectory_pairs = [
+        (
+            types.TrajectoryWithRew(
+                np.array([1, 2]),
+                np.array([1]),
+                np.array(
+                    [
+                        {
+                            "video_path": pathlib.Path(
+                                "tests/algorithms/test_preference_comparisons.py",
+                            ),
+                        },
+                    ],
+                ),
+                True,
+                np.array([1.0]),
+            ),
+            types.TrajectoryWithRew(
```

Comment:
Is this identical to above? Can we assign it to a temporary variable e.g. `traj` and then rewrite `trajectory_pairs = [(traj,traj)]`?